examples/Viewer.tsx

```diff
@@ -145,8 +143,9 @@ class Viewer extends React.Component<{}, ViewerState> {
     private viewerRef: React.RefObject<SimulariumViewer>;
     private panMode = false;
     private focusMode = true;
+    private netConnectionSettings: NetConnectionParams;
 
-    public constructor(props) {
+    public constructor(props: ViewerState & InputParams) {
```

Comment:
I think this can just be `InputParams`, as only those fields are used from `props`. This can also go in the first generic argument to `React.Component` above, which is for the props type: `class Viewer extends React.Component<InputParams, ViewerState> {`

Reply:
- [ ] I'll do that!